cmd/ausoceantv/plans.html

```diff
@@ -9,9 +9,9 @@
   </head>
   <body class="h-screen">
     <div class="container flex h-full items-center justify-center gap-5">
-      <plan-element plan-type="Day Pass" plan-cost="4" class="h-fit w-1/3" href="/checkout?product=day-pass"></plan-element>
-      <plan-element plan-type="Monthly Subscription" plan-cost="5" class="h-fit w-1/3" href="/checkout?product=month-subscription"></plan-element>
-      <plan-element plan-type="Yearly Subscription" plan-cost="30" comp-cost="60" class="h-fit w-1/3" href="/checkout?product=year-subscription"></plan-element>
+      <plan-element plan-type="Day Pass" plan-cost="4" class="h-fit w-1/3" href="/checkout.html?priceID=price_1QZMqMDMhXzQAv2TK0EFunPO"></plan-element>
```

Comment:
Maybe dumb question but could we have less obscure names for the priceIDs that then map them appropriately to the strip IDs ? Then you can have these IDs change (not sure why they would) and not have it break the frontend ?

Reply:
I think this would be following the “Thin Client, Thick Server” (or “Server-Driven UI”) principle more, if I'm actually understanding this.